assemblies/pentaho-war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -380,6 +387,11 @@
     <url-pattern>/webservices/*</url-pattern>
   </servlet-mapping>
 
+  <servlet-mapping>
+    <servlet-name>jaxwsPluginEndpoint-spring</servlet-name>
+    <url-pattern>/webservices/plugins/*</url-pattern>
+  </servlet-mapping>
```

Comment:
So, this adds the ability of plugins to have JAX-WS-based webservices.

While not a one-to-one replacement, have you considered the existing plugin-level Axis-based webservices?

https://hv-eng.atlassian.net/wiki/spaces/ENG/pages/30211178504/Web+Services#REST-%2F-SOAP-%5BhardBreak%5DApache-Axis2%5BhardBreak%5D%5BhardBreak%5D

Assuming this stays, ideally, this wiki page would be updated with the new web-service type for plugins.